src/balance.rs

```diff
@@ -233,3 +236,9 @@ where
         Ok(amount)
     }
 }
+
+fn bytevec2accountid<T: SysConfig>(vec: &Vec<u8>) -> Result<AccountIdOf<T>, DispatchError> {
```

Comment:
looks too tight without any underscore
```suggestion
fn vec_to_account_id<T: SysConfig>(vec: &Vec<u8>) -> Result<AccountIdOf<T>, DispatchError> {
```
or just try to implement TryFrom here if you have time

Reply:
Done